objcgen/Makefile

```diff
@@ -15,3 +15,29 @@ ifeq ($(SHELLCHECK),)
 else
 	$(SHELLCHECK) $^
 endif
+
+VERSION = 0.1
+FRAMEWORK_DIR = _build/Library/Frameworks/Xamarin.Embeddinator-4000
+INSTALLER_FILES = Xamarin.Embeddinator-4000-$(VERSION).pkg
+
+package:: bin/Debug/objcgen.exe
+	mkdir -p $(FRAMEWORK_DIR)/Commands/
+	mkdir -p $(FRAMEWORK_DIR)/Versions/$(VERSION)/bin
+	ln -fs $(VERSION) $(FRAMEWORK_DIR)/Versions/Current
+	cp bin/Debug/objcgen.exe $(FRAMEWORK_DIR)/Versions/Current/bin/
+	cp bin/Debug/Mono.Options.dll $(FRAMEWORK_DIR)/Versions/Current/bin/
+	cp script/objcgen  $(FRAMEWORK_DIR)/Commands/
+	chmod +x $(FRAMEWORK_DIR)/Commands/objcgen
+
+clean-package::
+	rm -rf _build/
+	rm -rf _installer/
+
+
+$(INSTALLER_FILES): package
+	rm -rf _installer/
+	mkdir -p _installer/ROOT/
+	cp -R _build/ _installer/ROOT/
+	pkgbuild --root _installer/ROOT --identifier com.xamarin.embeddinator-4000 --version 0.1 $(INSTALLER_FILES)
```

Comment:
and `$(VERSION)` should be reused here (because we'll always forget it otherwise)